public-dashboard-client/src/controls/CohortSelect.js

```diff
@@ -193,15 +210,26 @@ function CustomSelect({
                 {...itemProps}
                 aria-selected={isSelected}
                 as="li"
-                backgroundColor={isSelected ? opt.color : undefined}
+                backgroundColor={getOptionColor({
+                  highlightedIndex,
+                  index,
+                  isSelected,
+                  opt,
+                })}
+                // color in this menu does not change because we just fade the others instead
+                highlightColor={opt.color}
                 highlightedSelector={
                   highlightedIndex === index ? `&#${itemProps.id}` : undefined
                 }
                 key={opt.id}
               >
                 <MenuItemContents>
                   {opt.label}
-                  <MenuItemCheckMark src={checkMarkPath} />
+                  <MenuItemCheckMark
+                    // alt="checked"
+                    // aria-hidden
```

Comment:
👏

Reply:
ha i'm not sure what you're applauding? looks like I forgot to delete these lines that I commented out (or to uncomment them if they would still be useful; need to double-check that)

Reply:
sorry I was confused by this because it was actually from an earlier commit -- however it was a good catch because I really did need to have an `alt` attribute here! I set it to empty because it doesn't serve any purpose for a screen reader, it's just duplicating the "selected" state of the menu `option`.